src/heroes/sombra/init.opy

```diff
@@ -16,22 +16,22 @@ def initSombra():
     eventPlayer.hero_initialized = true
 
 
-rule "[sombra/init.opy]: Correct Sombra Melee damage":
-    @Event playerDealtDamage
-    @Hero sombra
-    @Condition eventAbility == Button.MELEE
+# rule "[sombra/init.opy]: Correct Sombra Melee damage":
+#     @Event playerDealtDamage
+#     @Hero sombra
+#     @Condition eventAbility == Button.MELEE
 
-    damage(victim, attacker, (eventDamage/eventPlayer._base_damage_scalar - eventDamage)/eventPlayer._base_damage_scalar)
+#     damage(victim, attacker, (eventDamage/eventPlayer._base_damage_scalar - eventDamage)/eventPlayer._base_damage_scalar)
 
 
 rule "[sombra/init.opy]: Correct Virus damage":
     @Event playerDealtDamage
     @Hero sombra
     @Condition eventAbility == Button.ABILITY_1
-    @Condition eventDamage >= ADJ_SOMBRA_VIRUS_IMPACT_DAMAGE
+    @Condition eventDamage >= OW2_SOMBRA_VIRUS_IMPACT_DAMAGE*eventPlayer._base_damage_scalar
 
     heal(victim, null, ((eventDamage * (ADJ_SOMBRA_VIRUS_IMPACT_DAMAGE/OW2_SOMBRA_VIRUS_IMPACT_DAMAGE) - eventDamage) * -1) * eventPlayer._base_damage_scalar)
```

Comment:
I feel like killing with virus impact damage isn't uncommon and will happen even more with buffed smg damage especially after emp.